mvpa2/clfs/warehouse.py

```diff
@@ -18,7 +18,7 @@
      MulticlassClassifier, RegressionAsClassifier
 from mvpa2.clfs.smlr import SMLR
 from mvpa2.clfs.knn import kNN
-from mvpa2.clfs.gda import LDA, QDA
+from mvpa2.clfs.gda import LDA, QDA, ShrinkageLDA
```

Comment:
after you move that definition into a dedicated file, you could place the import within `if externals.exists('skl')` below so it would be imported only when sklearn installed

Then add an actual instance of the class, similarly to how it is done for LDA, GDA etc in this file.  This should gets its testing going and we will continue from there